pulseapi/utility/management/commands/load_fake_data.py

```diff
@@ -35,6 +35,15 @@ def add_arguments(self, parser):
             help='A seed value to pass to Faker before generating data'
         )
 
+        parser.add_argument(
+            '--fellowsCount',
```

Comment:
Can you change to "fellows-count"?

Reply:
yup. Can you tell I'm used to PowerShell (Although, it would have been `-FellowsCount` if a Powershell utility)